src/core_plugins/kibana/public/management/sections/indices/add_data_steps/parse_csv_step/parse_csv_step.js

```diff
@@ -21,6 +22,8 @@ modules.get('apps/management')
         const maxSampleRows = 10;
         const maxSampleColumns = 20;
 
+        this.maxBytesFormatted = numeral(addDataMaxBytes).format('0 b');
```

Comment:
Doesn't seem this is used anywhere. Did you mean to use it in the formatted error message on line 63?